app/models/vacancy.rb

```diff
@@ -102,6 +102,24 @@ class Vacancy < ApplicationRecord
     end
 
     geoloc :lat, :lng
+
+    attributesForFaceting [:job_roles, :working_patterns, :school]
```

Comment:
We can configure these in the dashboard. Listing the attributes there has the advantage of being more easily configurable.

https://www.algolia.com/apps/QM2YE0HRBW/explorer/configuration/Vacancy/facets

<img width="671" alt="Screenshot 2020-05-12 at 10 40 43" src="https://user-images.githubusercontent.com/60350599/81668800-1685ba00-943d-11ea-8e08-1382d73b0028.png">

Reply:
Part of the thinking here was that we don't want someone to un-configure this by mistake

Reply:
Is it going to need to change very often (more than every couple of months)? If not, seems fine to have it here.